auth/src/commonMain/kotlin/com/liftric/AuthHandler.kt

```diff
@@ -206,7 +243,7 @@ open class AuthHandler(private val configuration: Configuration) : Auth {
                 )
                 body = payload
             }
-            return if (response.status.value == 200) {
+            return if (response.status == HttpStatusCode.OK) {
```

Comment:
see above (omitting return)

Reply:
A 'return' expression required in a function with a block body ('{...}')